zap/src/main/java/org/parosproxy/paros/core/scanner/Variant.java

```diff
@@ -47,6 +47,17 @@ public String setParameter(
     public String setEscapedParameter(
             HttpMessage msg, NameValuePair originalPair, String param, String value);
 
+    /**
+     * Sets the parameters into the given {@code message}.
+     *
+     * @param message the message that will be changed
+     * @param appParams list of name of the parameter
+     * @return the parameter values set to HttpMessage
+     */
+    default List<String> setParameters(HttpMessage message, List<AppParameter> appParams) {
```

Comment:
The return types are quite conflicting, like for multipart it is returning body of HttMessage where as for cookie it returns something else and for some variants it is returning the passed values.

Reply:
Fine returning void.